bolt_support.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "BOLT protocol support"
+nav_order: 10
+description: "Connecting FalkorDB using BOLT protocol."
+---
+
+## BOLT protocol support for FalkorDB
+
+FalkorDB provides support for querying using BOLT drivers. This guide will walk you through the process of connecting to FalkorDB using the BOLT protocol.
+
+### Prerequisites
+
+Before you begin, ensure that you have a FalkorDB instance up and running. You can use either Docker or the FalkorDB sandbox for this purpose. Additionally, install the necessary BOLT drivers:
+
+```bash
+pip install neo4j
+```
+
+### Step 1: Create a `main.py` File
+
+Create a main.py file with the following content:
+
+```python
+from neo4j import GraphDatabase
+
+driver = GraphDatabase.driver("bolt://localhost:7687", auth=("falkordb", ""))
```

Comment:
When running either a docker image of `FalkorDB` or when running in the cloud, what's the default credentials?
is it Username:'falkordb' and password:'' ?
If that's the case I think we need to change it, if not this documentation should indicate that the user creds' need to be specified, e.g. `<username>, <password>`